ACHNBrowserUI/ACHNBrowserUI/viewModels/DashboardViewModel.swift

```diff
@@ -27,6 +28,12 @@ class DashboardViewModel: ObservableObject {
             self?.fishes = items[.fish] ?? []
             self?.bugs = items[.bugs] ?? []
             self?.fossils = items[.fossils] ?? []
+            self?.art = []
+            items[.art]?.forEach { art in
```

Comment:
You can do it in one line by using a .filter on the [Item] art array. So something like `self?.art = items[.art].filter { } `. 
And instead of checking for the sell price I would check if the name contain "fake", this would be more robust in case the datasource change a bit in the future.